### PR TITLE
Andre lesson 1

### PR DESCRIPTION
**Pull request by AZhang2**
lesson 1

### CODE REVIEW

File: homework/lesson1/Lesson 1 homework-Andre.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+print("My name is Andre, and I am 14 years old")
+x = 14*12
```
Comment (Sadface1000) [inline homework/lesson1/Lesson 1 homework-Andre.py:2]: put it into int()

**Review by Sadface1000 — APPROVED** [review]

File: homework/Lesson 2 challenge-Andre.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+print("third order")
+print("please answer the following with numbers only:")
+pizzas3 = input("How many pizzas would you like? ")
+pepperonis3 = input("How many pepperonis do you want? ")
+olives3 = input("How many olives do you want? ")
+print(str(name) + "'s third order is " + str(pizzas3) + " pizzas with " + str(pepperonis3) + " pepperonis and " + str(olives3) + " olives.")
+average = print(str(name) + ''' ordered an average of:
+''' + str(float((int(pizzas1) + int(pizzas2) + int(pizzas3)) / 3)) + ''' pizzas with
+''' + str(float((int(pepperonis1) + int(pepperonis2) + int(pepperonis3)) / 3)) + ''' pepperonis and
+''' + str(float((int(olives1) + int(olives2) + int(olives3)) / 3)) + " olives per order.")
```
Comment (Shuzhengz) [inline homework/Lesson 2 challenge-Andre.py:28]: `'''` is only needed on the first and last line of the printed things, but this works too

File: homework/Lesson 3 homework-Andre.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+print('Is a ValueError because "4.5" is text type data put into a numbers function')
+print('int("4 and 5"):')
+print('Is a ValueError because again, there is text type data being put into a numbers function.')
+print(" ")
+print("1d")
+print('What happens when you try float("three point six") or float("three")?')
+print('float("three point six"):')
+print('Is a ValueError because text type data is being plugged into a numbers function')
+print('float ("three"):')
+print('Is a ValueError because text type data is being plugged into a numbers function')
```
Comment (Shuzhengz) [inline homework/Lesson 3 homework-Andre.py:24]: comments are what I had in mind but printing them out works as well

File: homework/Lesson 4 homework-Andre.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+elif int(age) >= 21 and int(age) <35:
+    print("You can stay alive, join the robotics team, drive, get a job, attend college, and be an adult.")
+elif int(age) >= 35 and int(age) < 50:
+    print("You can stay alive, join the robotics team, drive, get a job, attend college, be an adult, and beecome President.")
+elif int(age) >= 50 and int(age) < 150:
+    print("If its legal, just do your thing, BOOMER.")
+elif int(age) >= 150 and int(age) < 500:
+    print("You can go commit deathpacito.")
+else:
+    print("Bruh. Stop trying to troll.")
```
Comment (Shuzhengz) [inline homework/Lesson 4 homework-Andre.py:25]: lol
works but would prefer shorter code (some machines are not powerful enough so we always want to compress the code as much as possible).

**Comment by Shuzhengz** [comment]
address the changes if you want, you're cleared to merge.

File: homework/Lesson 2 challenge-Andre.py (hunk 2 above)
Comment (LZDCA) [reply homework/Lesson 2 challenge-Andre.py:28]: Tom, you are too fast. When I got in, it seems I don't need to do anyting:)

Comment (Shuzhengz) [reply homework/Lesson 2 challenge-Andre.py:28]: lol

**Pull request merged**